models.rdf

```diff
@@ -86,7 +86,16 @@
       <rdfs:comment xml:lang="en">Links to a File which is related to this Object but doesn't directly describe or represent it, such as technical metadata about other files.</rdfs:comment>
       <rdfs:domain rdf:resource="http://pcdm.org/models#Object"/>
       <rdfs:range rdf:resource="http://pcdm.org/models#File"/>
-      <rdfs:subPropertyOf rdf:resource="http://www.w3.org/ns/ldp#contains"/>
+      <rdfs:subPropertyOf rdf:resource="http://www.openarchives.org/ore/terms/aggregates"/>
+      <rdfs:isDefinedBy rdf:resource="http://pcdm.org/models#"/>
+    </rdf:Property>
+    
+    <rdf:Property rdf:about="http://pcdm.org/models#hasRelatedObject">
```

Comment:
I think having hasRelatedObject explicitly defined is better than using ore:aggregates directly (even if the domain and range are the same), since it lets us call out the specific semantics we intend.